frontend/system/src/Admin/Users/api/operations.graphql

```diff
@@ -0,0 +1,74 @@
+fragment UserAccountRow on UserAccountNode {
+  __typename
+  id
+  username
+  email
+  displayName
+  permissions
+  auditLogs {
```

Comment:
Can take this out if we're not displaying them?

Reply:
Great catch thanks [823a790](https://github.com/msupply-foundation/unified-codes/pull/454/commits/823a790facafe553f9c76ae3e681f36825ca06b4)